File: branches/simosws/simosws/BusinessLayer/DTO/Asignacion.vb
Public Class Asignacion


    Private _encuestaId As Integer
    Public Property EncuestaId() As Integer
        Get
            Return _encuestaId
        End Get
        Set(ByVal value As Integer)
            _encuestaId = value
        End Set
    End Property


    Private _establecimientoId As String
    Public Property EstablecimientoId() As String
        Get
            Return _establecimientoId
        End Get
        Set(ByVal value As String)
            _establecimientoId = value
        End Set
    End Property


    Private _establecimientoName As String
    Public Property EstablecimientoName() As String
        Get
            Return _establecimientoName
        End Get
        Set(ByVal value As String)
            _establecimientoName = value
        End Set
    End Property


    Private _fechaProgramacion As String
    Public Property FechaProgramacion() As String
        Get
            Return _fechaProgramacion
        End Get
        Set(ByVal value As String)
            _fechaProgramacion = value
        End Set
    End Property


    Private _turnoId As Integer
    Public Property TurnoId() As Integer
        Get
            Return _turnoId
        End Get
        Set(ByVal value As Integer)
            _turnoId = value
        End Set
    End Property


    Private _turnoDescripcion As String
    Public Property TurnoDescripcion() As String
        Get
            Return _turnoDescripcion
        End Get
        Set(ByVal value As String)
            _turnoDescripcion = value
        End Set
    End Property


    Private _fechaProgramada As String
    Public Property FechaProgramada() As String
        Get
            Return _fechaProgramada
        End Get
        Set(ByVal value As String)
            _fechaProgramada = value
        End Set
    End Property


End Class

File: branches/simos/src/gob/sis/simos/entity/Asignacion.java
package gob.sis.simos.entity;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

@DatabaseTable(tableName="eess")
public class Asignacion {
	
	@DatabaseField(columnName="encta_id",id=true)
	private Integer encuestaId;
	
	@DatabaseField(columnName="eess_id")
	private String establecimientoId;
	
	@DatabaseField(columnName="eess_name")
	private String establecimientoName;
	
	@DatabaseField(columnName="eess_desc")
	private String establecimientoDesc;
	
	public Integer getEncuestaId() {
		return encuestaId;
	}

	public void setEncuestaId(Integer encuestaId) {
		this.encuestaId = encuestaId;
	}

	public String getEstablecimientoId() {
		return establecimientoId;
	}

	public void setEstablecimientoId(String establecimientoId) {
		this.establecimientoId = establecimientoId;
	}

	public String getEstablecimientoName() {
		return establecimientoName;
	}

	public void setEstablecimientoName(String establecimientoName) {
		this.establecimientoName = establecimientoName;
	}

	public String getEstablecimientoDesc() {
		return establecimientoDesc;
	}

	public void setEstablecimientoDesc(String establecimientoDesc) {
		this.establecimientoDesc = establecimientoDesc;
	}


}
